src/CmfRoutingAutoBundle.php

```diff
@@ -50,7 +56,7 @@ private function buildPhpcrCompilerPass(ContainerBuilder $container)
                         realpath(__DIR__.'/Resources/config/doctrine-model') => 'Symfony\Cmf\Bundle\RoutingAutoBundle\Model',
                     ],
                     ['cmf_routing_auto.persistence.phpcr.manager_name'],
-                    false,
+                    'cmf_routing.backend_type_phpcr',
```

Comment:
what is the purpose of this change?

Reply:
it allows to disable the mapping of phpcr documents when you load auto routing orm bundle. You can see here https://github.com/wearemarketing/routing-auto-orm-bundle/blob/master/src/CmfRoutingAutoOrmBundle.php#L26

Reply:
can we not instead only add the PHPCR mappings if the PHPCR adapter is used?

Reply:
but in anycase, this looks like a good change cc @dbu

Reply:
Actually, it is a good change as well, I think it is so common to add that parameter to allow or not the mapping.

Reply:
yep, looks sensible to me.